cmd/litcli/helpers.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+	"fmt"
+	"strconv"
+
+	"github.com/lightninglabs/lightning-terminal/session"
+	"github.com/urfave/cli"
+)
+
+// helperCommands are commands that do not require a connection to LiTd to
+// return a response.
+var helperCommands = cli.Command{
+	Name:     "helper",
+	Usage:    "helper commands",
+	Category: "Helper",
+	Subcommands: []cli.Command{
+		generateSuperMacRootIDCmd,
+		isSuperMacaroonCmd,
+	},
+}
+
+var generateSuperMacRootIDCmd = cli.Command{
+	Name: "supermacrootkey",
+	Usage: "Generate a valid super macaroon root key ID from scratch " +
+		"or from a given root key ID suffix",
+	Description: `
+	This command can be used to generate a valid super macaroon root key ID
+	from scratch or from a given root key ID suffix.
+	`,
+	Action: superMacRootKey,
+	Flags: []cli.Flag{
+		cli.StringFlag{
```

Comment:
It's not really clear to me just from the text how to specify this. Should we allow this to be 4 bytes (=8 hex characters) so we can simplify the checks a bit?

Reply:
good point :) updated!